frontend/src/app/main/ui/workspace/tokens/form.cljs

```diff
@@ -156,30 +156,43 @@
 
 (defonce form-token-cache-atom (atom nil))
 
-(mf/defc ramp
+(defn hex->value
+  [hex]
+  (when-let [tc (tinycolor/valid-color hex)]
+    (let [hex (str "#" (tinycolor/->hex tc))
+          [r g b] (c/hex->rgb hex)
+          [h s v] (c/hex->hsv hex)]
+      {:hex hex
+       :r r :g g :b b
+       :h h :s s :v v
+       :alpha 1})))
+
+(mf/defc ramp*
   [{:keys [color on-change]}]
   (let [wrapper-node-ref (mf/use-ref nil)
-        dragging? (mf/use-state)
-        hex->value (fn [hex]
-                     (when-let [tc (tinycolor/valid-color hex)]
-                       (let [hex (str "#" (tinycolor/->hex tc))
-                             [r g b] (c/hex->rgb hex)
-                             [h s v] (c/hex->hsv hex)]
-                         {:hex hex
-                          :r r :g g :b b
-                          :h h :s s :v v
-                          :alpha 1})))
-        on-change' (fn [{:keys [hex]}]
-                     (when-not (and @dragging? hex)
-                       (on-change hex)))]
+        dragging-ref     (mf/use-ref false)
```

Comment:
The `mf/use-state` triggers a component redraw each time the state is changed. In this case this is not necessary, because the value is only used in other internal function.

We can use `mf/use-ref` that serves the same purpose, but doesn't triger redraws.